numpy/lib/tests/test_io.py

```diff
@@ -568,8 +568,9 @@ def test_unicode_and_bytes_fmt(self, fmt, iotype):
         else:
             assert_equal(s.read(), b"%f\n" % 1.)
 
-    @pytest.mark.skipif(sys.platform=='win32',
-                        reason="large files cause problems")
+    @pytest.mark.skipif(sys.platform=='win32', reason="files>4GB may not work")
+    @pytest.mark.skipif(IS_PYPY,
+         reason="GC problems after test, even after multiple gc.collect calls")
```

Comment:
Is this worth filing as an issue either in the pypy tracker or the numpy one, and then including the issue id in this reason?

Reply:
linked to gh-15775